clients/python/agentic-sandbox-client/sandbox-router/sandbox_router.py

```
@@ -210,4 +243,4 @@ async def stream_generator():
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)